app/controllers/ChargeSummaryController.scala

```diff
@@ -184,6 +186,23 @@ class ChargeSummaryController @Inject()(val auth: AuthenticatorPredicate,
 
   }
 
+  private def checkForPoaExtraChargeLink(chargeDetailsForTaxYear: FinancialDetailsModel, documentDetailWithDueDate: DocumentDetailWithDueDate, isAgent: Boolean): Option[String] = {
+    val desiredMainTransaction: String = documentDetailWithDueDate.documentDetail.getDocType match { //TODO: Replace with new mainTransaction method
```

Comment:
I would assume you are still working on this segment of code?

Reply:
This code will need to be refactored once Owen's ticket is merged and we can use mainTransaction to catergorise charges. I didn't realise I'd left the TODO in there though, will remove it

Reply:
This will need to be done for a couple other tickets as well, and I'm happy to do it, just didn't want it to hold everything up

Reply:
I think in this case we need to merge @owen-gilfellon  story first, as this is a bit redundant manipulation